.github/workflows/dev-build.yml

```diff
@@ -11,6 +11,24 @@ on:
       - .gitignore
 
 jobs:
+  get-previous-tag:
+    name: Get previous tag
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 100
+          fetch-tags: true
+      - uses: younited/get-previous-tag-action@v1.0.0
+        with:
+          match: "*.*.*-dev"
+      - id: previous-tag
```

Comment:
Why don't we use the result of the previous step? Do we have some problems with that action?

Reply:
Yes, get-previous-tag-action plugin found 2.2.0 instead of 2.2.1 tag

Reply:
Got it. Then we, probably, should remove `younited/get-previous-tag-action@v1.0.0` from steps

Reply:
Yes, I will. I added it to show the problem in actions log

Reply:
We also have other actions for GitHub. Like this one:
https://github.com/WyriHaximus/github-action-get-previous-tag

https://github.com/marketplace?category=&query=latest+tag+sort%3Apopularity-desc&type=actions&verification=

Reply:
[github-action-get-previous-tag](https://github.com/WyriHaximus/github-action-get-previous-tag) doesn't support pattern, prefix only.
I think we can move the current action implementation into .github in future.